tests/integration_test/services_test.py

```
@@ -38,6 +38,14 @@
 from kairon.shared.chat.data_objects import Channels
 from kairon.shared.content_importer.content_processor import ContentImporterLogProcessor
 from kairon.shared.importer.data_objects import ValidationLogs
+from kairon.shared.log_system.base import BaseLogHandler
```

Ruff (0.12.2)

`kairon.shared.log_system.base.BaseLogHandler` imported but unused

Remove unused import: `kairon.shared.log_system.base.BaseLogHandler`

(F401)